geode-core/src/main/java/org/apache/geode/security/ResourcePermission.java

```diff
@@ -40,7 +40,7 @@
 
   // when ALL is specified, we need it to convert to string "*" instead of "ALL".
   public enum Target {
-    ALL(ResourcePermission.ALL), DISK, GATEWAY, QUERY, JAR;
+    ALL(ResourcePermission.ALL), DISK, GATEWAY, QUERY, DEPLOY;
```

Comment:
I think this is a breaking change to a public API.  Can we change the value of the enum element but leave its name as is for backwards compatibility?  I.e. `JAR ("DEPLOY")`.

Reply:
Swap said this is not released yet. We can still change it.